.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: daily
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
I think weekly ought to be enough for both of these... maybe even monthly.

Reply:
```suggestion
    interval: weekly
```